plugins/mcreator-core/procedures/blockat_plant_type.json

```diff
@@ -66,7 +35,7 @@
       "z"
     ],
     "fields": [
-      "type"
+      "planttype"

```

Comment:
Any reason why you did not leave this at "type" name? In such a case, existing procedures using this block will remain operational I think.

Reply:
It's because I used "planttype" in `BlocklyJavascriptBridge`, so it doesn't recognize the field if I left it "type". Imo the better name is more important; since this procedure isn't used often and was added the last snapshot, it shouldn't cause too many issues.

Reply:
Fair point ;)